File: elixir/fizzbuzz/test/fizzbuzz_test.exs
defmodule FizzbuzzTest do
  use ExUnit.Case
  doctest Fizzbuzz

  test "takes 3 then returns Fizz" do
    assert Fizzbuzz.say(3) == :Fizz
  end

  test "takes 6 then returns Fizz" do
    assert Fizzbuzz.say(6) == :Fizz
  end

  test "takes 5 then returns Buzz" do
    assert Fizzbuzz.say(5) == :Buzz
  end

  test "takes 10 then returns Buzz" do
    assert Fizzbuzz.say(10) == :Buzz
  end

  test "takes 15 then returns FizzBuzz" do
    assert Fizzbuzz.say(15) == :FizzBuzz
  end

  test "takes 30 then returns FizzBuzz" do
    assert Fizzbuzz.say(30) == :FizzBuzz
  end
end

File: java/fizzbuzz/src/test/java/FizzBuzzTest.java
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class FizzBuzzTest {
    @Test
    void testGetOneShouldReturnOne() {
        FizzBuzz fizzbuzz = new FizzBuzz();
        String actual = fizzbuzz.get(1);
        Assertions.assertEquals("1", actual);
    }
}